dispatch/static/manager/src/js/actions/UserActions.js

```diff
@@ -35,6 +35,23 @@ export function authenticateUser(email, password, nextPath = '/') {
   }
 }
 
+export function verifyToken(token) {
+  return (dispatch) => {
+    dispatch({ type: pending(types.AUTH.VERIFY_TOKEN )})
+
+    return DispatchAPI.auth.verifyToken(token)
+      .then(() => {
+        dispatch({
+          type: fulfilled(types.AUTH.VERIFY_TOKEN)
+        })
```

Comment:
Include an `isValid` attribute in the action that can be used by the reducer to delete the cookie if necessary